addons/website_twitter/static/src/js/website.twitter.editor.js

```diff
@@ -13,35 +13,37 @@ sOptions.registry.twitter = sOptions.Class.extend({
      */
     start: function () {
         var self = this;
-        var $configuration = dom.renderButton({
+        this.$configuration = dom.renderButton({
             attrs: {
                 class: 'btn-primary d-none',
                 contenteditable: 'false',
             },
             text: _t("Reload"),
         });
-        $configuration.appendTo(document.body).on('click', function (ev) {
+        this.$configuration.appendTo(document.body).on('click', function (ev) {
```

Comment:
Would not it be easier to put the button inside of the snippet instead of the body? That way you would solve the removal problem, probably solve the strange scrollbar which appears on hover and maybe simplify some code which is probably dedicated to position the button correctly with JS?

Reply:
I wanted to keep a minimal diff in stable, but I do agree this is a better solution.